docs/configuration-reference/backend/local.md

```diff
@@ -0,0 +1,43 @@
+# Local backend configuration reference
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Configuration](#configuration)
+* [Argument reference](#argument-reference)
+
+## Introduction
+
+Lokomotive supports local backend for storing Terraform state.
```

Comment:
I think we should have 3 small docs. One for backend block configuration, one for local and one for s3. then backend block configuration document should just refer to the available types and their documentation.

This way, the hierarchy is nicely represented.

Reply:
the third document (which then refers to local and s3) would be very small only a few lines.

Having the title `local.md` and `s3.md` under backend  directory I think suffices the hierarchy.

Reply:
:+1: I won't push on that, It's good enough what we have right now.